src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(BallCollors.values().length);
+        BallCollors color = BallCollors.values()[index];
+        return color.toString();
```

Comment:
Check common mistake
```suggestion
        return color.name().toLowerCase();
```